frontend/changelog.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
+
+## [1.0.0] - 2022-05-04
```

Comment:
don't forget to improve version in packaqe.json and package-lock.json too

Reply:
version 1.1.0

Reply:
it will be [Unreleased]